public/app/features/transformers/editors/GroupingToMatrixTransformerEditor.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useCallback } from 'react';
+import {
+  DataTransformerID,
+  SelectableValue,
+  standardTransformers,
+  TransformerRegistryItem,
+  TransformerUIProps,
+} from '@grafana/data';
+import { InlineField, InlineFieldRow, Select } from '@grafana/ui';
+import { GroupingToMatrixTransformerOptions } from '@grafana/data/src/transformations/transformers/groupingToMatrix';
```

Comment:
instead of importing from a subpath, can you expose this method in the `@grafana/data` index the same way as valueMatchers? https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/index.ts#L23

```suggestion
import { GroupingToMatrixTransformerOptions } from '@grafana/data';
```

Reply:
Sure thing, seems a bit incosistent with other transformers tho

Reply:
mmmm agreed, there's a bit of inconsistency here. it doesn't really matter in the end as they both work. my gut is that anything we want to export from `@grafana/data` should be exposed on the index, then the public interface is clear. wonder if @ryantxu has an opinion? 😄